configuration.php

```diff
@@ -4,6 +4,8 @@
 use Icinga\Module\Nagvis\RestrictionHelper;
 
 $section = $this->menuSection(N_('Maps'))
+    ->setIcon('globe')
+    ->add($this->translate('NagVis'))
```

Comment:
Please note that you are switching context here. After this, $section is no longer the main menu entry but a sub-menu. Following code makes then wrong assumptions and creates sub-sub-menus. While in theory we support such, there are no explicitly specified rendering rules for them. So this change has the potential to break things.